tests/cypress/tests-e2e/search/search-results.cy.js

```diff
@@ -392,6 +392,11 @@ describe('Test the search bar result handling', () => {
         cy.get(searchbarSelector).click()
         cy.get('@locationSearchResults').should('not.be.visible')
 
+        cy.log('Testing layer feature search')
+        cy.get(searchbarSelector).click()
+        cy.get('[data-cy="search-results-featuresearch"] [data-cy="search-result-entry"]').click()
+        cy.url().should('contain', 'center=2566639.39,1205531.89')
```

Comment:
can you create the expected outcome from `layerFeatureResponse` instead (and more sepcifically `expectedCenterDefaultProjection`)? This would make it more robust for later changes